### PR TITLE
Topic/international variants table

### PR DESCRIPTION
**Pull request by VWSCoronaDashboard1**
## Summary

Serverside data shaping for the variants

### Governance

- [ ] Documentation is added
- [ ] Test cases are added or updated
- [ ] I've read the contributing document https://github.com/minvws/.github/blob/master/CONTRIBUTING.md
- [ ] I've read and understand the Code of Conduct https://github.com/minvws/.github/blob/master/CODE_OF_CONDUCT.md
- [ ] I understand that any contributions or suggestions I made may make it into the actual code. I've read the License
      https://github.com/minvws/nl-covid19-notification-app-coordination/blob/master/LICENSE.txt
      and the contributor license agreement https://github.com/minvws/nl-covid19-notification-app-coordination/blob/master/CLA.md

### CODE REVIEW

File: packages/app/src/domain/variants/variants-table-tile/index.tsx [hunk 1]
```diff
@@ -47,19 +45,26 @@ export function VariantsTableTile({
       <Box maxWidth="maxWidthText">
         <Markdown
           content={replaceVariablesInText(text.varianten_tabel.omschrijving, {
-            sample_size: data.sample_size,
+            sample_size: '',
```
Comment (VWSCoronaDashboard8) [inline packages/app/src/domain/variants/variants-table-tile/index.tsx:48]: Why not remove the property is sample_size is not used in the text? Or maybe add a comment to explain why you're using an empty string.

File: packages/app/src/static-props/variants/get-variant-chart-data.ts [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+
+export function getVariantChartData(
+  nlVariants: NlVariants | undefined,
+  variantTranslations: SiteText['covid_varianten']['varianten']
+) {
+  if (!isDefined(nlVariants) || !isDefined(nlVariants.values)) {
+    return EMPTY_VALUES;
+  }
+
+  const vocVariants = nlVariants.values.filter(
```
Comment (VWSCoronaDashboard8) [inline packages/app/src/static-props/variants/get-variant-chart-data.ts:31]: nit; you can avoid the confusing acronym by calling it `variantsOfConcern`. It's not many more characters and it reads better IMO.

File: packages/app/src/static-props/variants/get-variant-chart-data.ts [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+    .filter((x) => x.endsWith('_percentage'))
+    .map((x) => x.slice(0, x.indexOf('_')));
+
+  const seriesConfig = variantNames.map<
+    LineSeriesDefinition<VariantChartValue>
+  >((x) => {
+    const color = (colors.data.variants as Record<string, string>)[x];
+    const label = (variantTranslations as Record<string, string>)[x];
+    assert(color, `No color specified for variant called "${x}"`);
+    assert(label, `No label specified for variant called "${x}"`);
```
Comment (VWSCoronaDashboard8) [inline packages/app/src/static-props/variants/get-variant-chart-data.ts:66]: nit; The assertions state that these can be undefined so the used types are not really correct. For these of cases I sometimes define a `Dictionary` type:

```
type Dictionary<T> = Record<string, T | undefined>
```

```suggestion
    const color = (colors.data.variants as Dictionary<string>)[x];
    const label = (variantTranslations as Dictionary<string>)[x];
    assert(color, `No color specified for variant called "${x}"`);
    assert(label, `No label specified for variant called "${x}"`);
```

**Review by VWSCoronaDashboard8 — APPROVED** [review]

File: packages/app/src/domain/variants/variants-table-tile/index.tsx (hunk 1 above)
Comment (VWSCoronaDashboard1) [reply packages/app/src/domain/variants/variants-table-tile/index.tsx:47]: Oh, I forgot to add that one, this is a mistake, it should have a valid value. I'll fix it.

Comment (VWSCoronaDashboard1) [reply packages/app/src/domain/variants/variants-table-tile/index.tsx]: Fixed

**Review by VWSCoronaDashboard14 — APPROVED** [review]

**Pull request merged**